control/autonomous_emergency_braking/include/autonomous_emergency_braking/node.hpp

```diff
@@ -65,6 +67,33 @@ using visualization_msgs::msg::MarkerArray;
 using Path = std::vector<geometry_msgs::msg::Pose>;
 using Vector3 = geometry_msgs::msg::Vector3;
 
+// For debugging
+class DebugTimer
```

Comment:
We have tier4_autoware_utils::StopWatch for this

Reply:
Deleted